snapcraft/internal/meta.py

```diff
@@ -29,7 +29,10 @@
 
 from snapcraft import file_utils
 from snapcraft import shell_utils
-from snapcraft.internal import common
+from snapcraft.internal import (
+    common,
+    repo
+)
```

Comment:
that really depends on who is looking :wink:

I personally find the use of parens ugly :confounded: